### DISCUSSION TITLE
Change the number of particles and radius of particles for the fluid model

### DISCUSSION THREAD

**Discussion by ytimber** [InteractiveComputerGraphics/SPlisHSPlasH#141] [Q&A] [answered]
Hi,

I have a naive question. How can I change the count of particles and radius of particles for the fluid model, such as the bunny? Also, I noticed that we can change the number of particles for emitter. Will we be able to change the radius of the particles for the emitter? If so, may you please tell me how? 
Many thanks in advance for your help!

**Comment by janbender** [comment, marked as answer]
Yes, you can change the radius. The radius cannot be changed during runtime. You have to do this in the scene file:

https://splishsplash.readthedocs.io/en/latest/file_format.html#simulation

Just set the attribute "particleRadius". 

You can also do it for the bunny. However, the bunny file (bgeo) is a pre-sampled file. This means only the particles get smaller, so the sampling won't be tight anymore. But you could also use any closed OBJ file with a mesh. Then the simulator will do a sampling at the beginning of the simulation with the defined radius. 

The radius is the same for all particles in the simulation, so a change of the particle radius also affects the size of emitted particles.

**Reply by ytimber** [reply]
@janbender Thank you so much with this detailed explanation. I just tried and it works! :))
